src/components/migrationToV3.stories.mdx

```diff
@@ -579,11 +718,11 @@ const App = () => (
 );
 
 const MyChild = () => {
-  const selectVideoInput = useSelectVideoInputDevice();
+  const meetingManager = useMeetingManager();
   const { selectDeviceError } = useVideoInputs();
 
   const handleClick = async (): => {
-    await selectVideoInput('deviceId');
+    await meetingManager.selectVideoInputDevice('deviceId');
```

Comment:
Do we need to change the Before example here?

Reply:
I don't think so, what change are you talking about?

Reply:
Discussed offline, updated